cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -1129,6 +1131,12 @@ def add_arguments(self, parser):
             help="Add items to solr as we create opinions. "
             "Items are not searchable unless flag is raised.",
         )
+        parser.add_argument(
+            "--bankruptcy-cases",
+            action="store_true",
+            help="Tells function to use bankruptcy courts for bankruptcy "
+            "cases.",
+        )
```

Comment:
I think the flag should be `--bankruptcy`

Reply:
I think it reads a little clearer and matches to what we've done elsewhere, but thats more taste